tests/findex.test.ts

```diff
@@ -241,17 +269,104 @@ test("SQLite", async () => {
       })
     }
   }
+  const upsertCallback = async (
+    table: string,
+    uidsAndValues: UidsAndValuesToUpsert,
+  ): Promise<UidsAndValues> => {
+    const rejected = [] as UidsAndValues
+    await Promise.all(
+      uidsAndValues.map(async ({ uid, oldValue, newValue }) => {
+        let changed
```

Comment:
should we initialize it to false?

Reply:
No since it's set in both branches of the condition below, Typescript will warn us if one of the branch doesn't set the value (it would no be the case if we init it with `false`)